core/lib/config/Cargo.toml

```diff
@@ -11,6 +11,7 @@ categories = ["cryptography"]
 
 [dependencies]
 zksync_basic_types = { path = "../../lib/basic_types" }
+zksync_types = { path = "../../lib/types" }
```

Comment:
`zksync_types` is a very heavyweight (and dangerous, w.r.t. the vast amount of exposed APIs) crate, so I'd prefer to not depend on it. Instead, you can define your own mode here, and implement the conversion between types where required manually. It'll be slightly more boilerplate, but it'd be safer.